### PR TITLE
add: link to GH on header and per page

### LINKED ISSUE #40: Add a link to GitHub
We don't have any links to this GitHub on the website right now. Would be great to get at least one link (maybe in the header bar?) that takes you here. Even better if we can add something like an "edit this page on github" link that would be present on each page.

### PR DESCRIPTION
**Pull request by platocrat**
Fixes #40 .

### CODE REVIEW

File: src/.vuepress/components/Home.vue [hunk 1]
```diff
@@ -8,141 +8,192 @@
...
       <a class="action-button" href="/docs">read the docs</a>
       <a class="action-button" href="/faqs">check out the FAQs</a>
-      <a class="action-button" href="https://discord.com/invite/jrnFEvq">join the discord</a>
+      <a class="action-button" href="https://discord.com/invite/jrnFEvq"
+        >join the discord</a
+      >
 
       <h3>
-        This is a work in progress. We'll be updating this site regularly in the coming weeks. Stay tuned!
+        This is a work in progress. We'll be updating this site regularly in the
```
Comment (smartcontracts) [inline src/.vuepress/components/Home.vue:25]: Is this auto-linting? Or manual changes?

File: src/.vuepress/config.js [hunk 2]
```diff
@@ -11,10 +11,13 @@ module.exports = {
...
-    editLinks: false,
-    docsDir: '',
-    editLinkText: '',
+    repo: 'https://github.com/ethereum-optimism/community-hub',
+    repoLabel: 'Edit on GitHub!',
+    editLinks: true,
+    docsDir: 'src',
+    docsRepo: 'https://github.com/ethereum-optimism/community-hub',
+    docsBranch: 'experimental',
+    editLinkText: 'Help us improve this page!',
```
Comment (smartcontracts) [inline src/.vuepress/config.js:20]:
```suggestion
    editLinkText: 'Something off? Suggest an edit to this page on GitHub by clicking here ✏️',
```

File: src/.vuepress/config.js [hunk 3]
```diff
@@ -11,10 +11,13 @@ module.exports = {
   ],
 
   themeConfig: {
-    repo: '',
-    editLinks: false,
-    docsDir: '',
-    editLinkText: '',
+    repo: 'https://github.com/ethereum-optimism/community-hub',
+    repoLabel: 'Edit on GitHub!',
```
Comment (smartcontracts) [inline src/.vuepress/config.js:15]:
```suggestion
    repoLabel: 'GitHub',
```

Comment (smartcontracts) [reply src/.vuepress/config.js:15]: Just for consistency

**Comment by smartcontracts** [comment]
@platocrat this is awesome, thank you! I didn't know this already existed in vuepress :-)

File: src/.vuepress/components/Home.vue (hunk 1 above)
Comment (platocrat) [reply src/.vuepress/components/Home.vue:25]: Auto-linting. First time using Vue, so I downloaded a VScode plugin that auto-lints 😅

**Comment by platocrat** [comment]
> @platocrat this is awesome, thank you! I didn't know this already existed in vuepress :-)

me neither 😂

File: src/.vuepress/components/Home.vue (hunk 1 above)
Comment (smartcontracts) [reply src/.vuepress/components/Home.vue:25]: Nice! No problem then.

**Review by smartcontracts — APPROVED** [review]
LGTM

**Pull request merged**